CONTRIBUTING.md

```diff
@@ -52,6 +56,7 @@ ensure that your code complies with some standard style conventions.
   ```
   - Make sure that the tests pass
   ```
+  python setup.py install
   python setup.py test
```

Comment:
there is also a *good practices* blog post on `pytest`: https://pytest.org/latest/goodpractices.html